app/src/main/res/values/booleans.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources>
-  <bool name="is_tablet">false</bool>
+  <bool name="is_tablet">true</bool>
```

Comment:
Rather than setting this to true, I'd remove it, and all references and methods using it.